docs/html/installation.md

```diff
@@ -67,9 +67,17 @@ $ python pip.pyz --help
 
 If run directly:
 
-```{pip-cli}
-$ pip.pyz --help
+````{tab} Unix/macOS
+```console
+$ ./pip.pyz
+```
```

Comment:
With the caveat that the users needs to do `chmod +x pip.pyz` after downloading. But I assume our docs is not the place to explain that.

Reply:
```suggestion
```
```

Reply:
I added it, let me know if I shouldn't have.